set_speed:
  header: 0x3E
  length: 0x08
  data: [0xA2, 0x00, 0x00, 0x00, 0x10, 0x27, 0x00, 0x00]

set_position:
  header: 0x3E
  length: 0x08
  data: [0xA4, 0x00, 0xF4, 0x01, 0xA0, 0x8C, 0x10, 0x00]

set_torque:
  header: 0x3E
  length: 0x08
  data: [0xA1, 0x00, 0x00, 0x00, 0x64, 0x00, 0x00, 0x00]

read_home_position:
  header: 0x3E
  length: 0x08
  data: [0x61, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00]
  
read_position:
  header: 0x3E
  length: 0x08
  data: [0x60, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00]

status_error:
  header: 0x3E
  length: 0x08
  data: [0x9A, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00]
  
status_motor:
  header: 0x3E
  length: 0x08
  data: [0x9C, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00]

stop_motor:
  header: 0x3E
  length: 0x08
  data: [0x81, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00]


